src/components/header.jsx

```diff
@@ -7,12 +7,15 @@ import SignIn from './SignIn'
 function Header () {
 
 
-  return (
-    <>
-      <SignIn/>
-      <div className="header"><GiQuillInk /> Scriptly </div>
-    </>
-  );
+    return (
+      <>
```

Comment:
@akb3y Not sure if you need these fragments anymore since you added the `div.header`. Just pointing out.